mu4/scenes/palette/qml/MuseScore/Palette/MoreElementsPopup.qml

```diff
@@ -150,7 +150,7 @@ StyledPopup {
             width: parent.width
             height: childrenRect.height
             border { width: 1; color: "black" }
-            color: ui.theme.backgroundColor //! TODO mscore.paletteBackground
+            color: ui.theme.backgroundPrimaryColor //! TODO mscore.paletteBackground
```

Comment:
Is this todo actual?

Reply:
https://github.com/musescore/MuseScore/pull/6471#discussion_r473030881